src/DevTools/setupTestWrapper.tsx

```diff
@@ -126,9 +126,11 @@ export const setupTestWrapperTL = <T extends OperationType>({
           />
         </MockBoot>
       ) : (
-        <RelayEnvironmentProvider environment={env}>
-          <Component {...componentProps} />
-        </RelayEnvironmentProvider>
+        <MockBoot relayEnvironment={env}>
+          <RelayEnvironmentProvider environment={env}>
+            <Component {...componentProps} />
+          </RelayEnvironmentProvider>
+        </MockBoot>
```

Comment:
I get that `MockBoot` was added so that all standard wrappers are present (esp the Palette theme which was specifically affecting these tests)

I'm still hazy on why _both_ `MockBoot` and `RelayEnvironmentProvider` are needed. (The env passed into `MockBoot` eventually makes its way to `Boot` → `SystemContext` but that is not sufficient I take it?)

Reply:
Hi Roop, made a video on why I believe this makes sense here
https://www.loom.com/share/c19184b6446e4970826487e8413a8aa7?sid=64deccb3-9cb3-4ca0-a0f4-95eb05745fea

Reply:
Excellent, thank you

Reply:
This was really helpful in explaining things, thanks @MounirDhahri. 

Makes sense in why this hasn't come up before, due to the fact that we're just now starting to adopt relay hooks in Force. In all of our other (more modern) apps, this provider is there, and one of the first things setup when we lay out the framework of the app. 

Couple things:
- Can we move this `RelayEnvironmentProvider` into MockBoot? I suspect this will come up in the future, and having it available to all of our code would be good
- Can we also add this to `Boot`?

Reply:
we actually spoke about this but we were unable to evaluate properly its impact. I think migrating this to the Mockboot would be the idea solution. 
Adding this to Boot as well sounds good as long as we make sure we use the same environment in our query renderers, I can PR that.

Reply:
I think it'll be safe 👍  We pluck the env from the system context everywhere for query renderers, and if we share the same env with the relay provider it (should) be backwards compatible.

Reply:
I'm going to rebase this on `main` and merge soon. I propose that we move any further test infra changes to a separate PR 🙏🏽

Reply:
Roger that 👍